src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -916,5 +916,10 @@ public enum NuGetLogCode
         /// InvalidUndottedFrameworkWarning
         /// </summary>
         NU5501 = 5501,
+
+        /// <summary>
+        /// No metadata file found
+        /// </summary>
+        NU5502 = 5502,
```

Comment:
I don't think this is the appropriate log code. 
Please refer to the top of the file for guidance on log codes.

Reply:
I did, went for `NU1110` but honestly not sure this is OK for this feature.